dhis-2/dhis-test-web-api/src/test/java/org/hisp/dhis/webapi/controller/tracker/export/singleevent/SingleEventsExportControllerH2Test.java

```diff
@@ -67,15 +66,6 @@
 class SingleEventsExportControllerH2Test extends H2ControllerIntegrationTestBase {
```

Comment:
Why do need this test? 🤔

Reply:
We actually don't need this test to be duplicated.
I moved it to the common package and just use one of the services.

Reply:
Why do we need the test at all?

Reply:
To test that the `content-type` is correct in the header? :man_shrugging: